extensions/cms/src/cmsResource/tree/cmsResourceTreeNode.ts

```diff
@@ -77,8 +85,11 @@ export class CmsResourceTreeNode extends CmsResourceTreeNodeBase {
 					}
 				}
 			}, (error) => {
```

Comment:
It's better practice and cleaner to use .catch at the end of the promise chain instead of having error handlers. It's more important for chains with multiple steps since the catch will catch any errors that occur along the way but it still makes it easier to read and modify later if you use .catch now

Reply:
`.catch(handler)` is literally just a wrapper for `.then(undefined, handler)`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch